content/_snippets/conf-file-location.md

```diff
@@ -0,0 +1 @@
+The [recommended installation](install-rippled.html) uses the config file `/etc/opt/ripple/rippled.cfg` by default. Other places you can put a config file include `$HOME/.config/ripple/rippled.cfg` (where `$HOME` is the home directory of the user running `rippled`), `$HOME/.local/ripple/rippled.cfg`, or the current working directory from where you start `rippled`.
```

Comment:
THIS! Thank you for writing this up -- the Additional Configuration section in the post-rippled-install.md snippet needs this. That section currently lists the location of rippled.cfg for yum only -- but I think we should list this instead - all of the valid locations? Separate ticket and PR?